python_modules/libraries/dagster-components/setup.py

```diff
@@ -50,7 +50,7 @@ def get_version() -> str:
     },
     extras_require={
         "sling": ["dagster-embedded-elt"],
-        "dbt": ["dagster-dbt"],
+        "dbt": ["dagster-dbt", "dagster-embedded-elt"],
```

Comment:
why does dbt require embedded-elt?

Reply:
https://github.com/dagster-io/dagster/blob/master/python_modules/libraries/dagster-components/dagster_components/lib/dbt_project.py#L11

Reply:
Yeah bad include. Its including AssetExecutionContext from the wrong spot